awscodecommit/Dockerfile

```diff
@@ -1,25 +1,31 @@
-FROM golang:alpine AS builder
+FROM golang:1.13-alpine AS builder
```

Comment:
pining go version to make builds more deterministic

Reply:
Just curious, why not 1.14?

Reply:
Didn't know that 1.14 was released when i was updating sources 

I think nothing bad will happen if we change version to 1.14, but in another PR

Reply:
Fine with me. The current `alpine` tag is an alias for `1.14.1-alpine3.11` by the way (https://hub.docker.com/_/golang) so depending on when the last build occurred, we may already be using 1.14.

Reply:
i don't mind updating the golang version in this PR.

Reply:
done